lib/active_fedora/associations/builder/directly_contains_one.rb

```diff
@@ -0,0 +1,39 @@
+module ActiveFedora::Associations::Builder
+  class DirectlyContainsOne < SingularAssociation #:nodoc:
+    self.macro = :directly_contains_one
+    self.valid_options += [:has_member_relation, :is_member_of_relation, :type, :through]
+    self.valid_options -= [:predicate]
+
+    def validate_options
+      if options[:through]
+        inherit_options_from_association(options[:through])
+      end
+      super
+
+      if !options[:has_member_relation] && !options[:is_member_of_relation]
+        raise ArgumentError, "You must specify a predicate for #{name}"
```

Comment:
This error message is a bit misleading. Can is say, specify has_member_relation or is_member_of_relation?